tests/appsec/test_processor.py

```diff
@@ -119,11 +119,17 @@ def test_headers_collection(tracer):
                 "accept": "something",
                 "x-Forwarded-for": "127.0.0.1",
             },
+            response_headers={
+                "foo": "bar",
+                "Content-Length": "500",
+            },
         )
 
     assert span.get_tag("http.request.headers.hello") is None
-    assert span.get_tag("http.request.headers.accept") == "something"
-    assert span.get_tag("http.request.headers.x-forwarded-for") == "127.0.0.1"
+    assert span.get_tag("http.request.headers.accept") == "['something']"
```

Comment:
this will be a breaking change for any user who currently relies on the value not being an array.


e.g. if they have any filters/queries/span2metrics/retention filters/etc that use `@http.request.headers.accept: something`

Reply:
Is this value going to be rendered consistently across all language implementations?

e.g. if we have as `['something']` but js has as `["something"]`, or java has as `something,another,last`

Reply:
Good point! I assumed the tags would be processed in the backend, but not being the case, I'll revert that.